manifests/job.pp

```diff
@@ -43,6 +44,9 @@
       jobname => $jobname,
     }
   } else {
+    if $config == undef and $source == undef and $template == undef {
+      fail('You should at least specify one of the $config, $source or $template param')
+    }
```

Comment:
That's quite strange to have a mandatory parameter not used when the job is disabled